### ISSUE TITLE
Unable to yarn install from scaffold-eth directory on Windows 11 OS

### ISSUE THREAD

**Issue by MetaRaj** [scaffold-eth/scaffold-eth#680]
Hi All, 
Intention is to clone scaffold-eth: installed latest versions of node.js(v17.5.0), git and yarn

I did yarn install on my /users/(youruser) and got yarn version 3.1.1
Then I cloned scaffold-eth from git
Then changed dir to scaffold-eth and did yarn install. This is where I failed to install yarn!!

Kindly help as I would like to learn scaffold-eth for hackathon submissions
Thank you very much.

The error output I got is as below:

C:\Users\(youruser)\scaffold-eth>yarn install

➤ YN0000: ┌ Project validation
➤ YN0057: │ @scaffold-eth/monorepo: 'nohoist' is deprecated, please use 'installConfig.hoistingLimits' instead
➤ YN0000: └ Completed
➤ YN0000: ┌ Resolution step
➤ YN0061: │ babel-eslint@npm:10.1.0 is deprecated: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
➤ YN0016: │ @nomiclabs/hardhat-ethers@npm:0.3.0-beta.10: Registry failed to return reference "0.3.0-beta.10"
➤ YN0000: └ Completed in 4s 390ms
➤ YN0000: Failed with errors in 4s 397ms
➤ YN0032: fsevents@npm:2.3.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: request@npm:2.88.2 is deprecated: request has been deprecated, see https://github.com/request/request/issues/3142
➤ YN0061: chokidar@npm:2.1.8 is deprecated: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies
➤ YN0061: fsevents@npm:1.2.13 is deprecated: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
➤ YN0061: eth-sig-util@npm:2.5.4 is deprecated: Deprecated in favor of '@metamask/eth-sig-util'
➤ YN0061: debug@npm:3.2.6 is deprecated: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
➤ YN0032: fsevents@npm:2.1.3: Implicit dependencies on node-gyp are discouraged
➤ YN0061: fsevents@npm:2.1.3 is deprecated: "Please update to latest v2.3 or v2.2"
➤ YN0061: uuid@npm:3.4.0 is deprecated: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
➤ YN0061: uuid@npm:2.0.1 is deprecated: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
➤ YN0061: uuid@npm:3.3.2 is deprecated: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
➤ YN0061: eth-sig-util@npm:1.4.2 is deprecated: Deprecated in favor of '@metamask/eth-sig-util'
➤ YN0013: ethereumjs-abi@https://github.com/ethereumjs/ethereumjs-abi.git#commit=ee3994657fa7a427238e6ba92a84d0b529bbcd
➤ YN0061: @walletconnect/types@npm:1.5.2 is deprecated: All published versioned below 1.6.0 are deprecated. Please upgrade to the latest version
➤ YN0013: concat-stream@https://github.com/hugomrdias/concat-stream.git#commit=057bc7b5d6d8df26c8cf00a3f151b6721a0a8034
➤ YN0013: ndjson@https://github.com/hugomrdias/ndjson.git#commit=4db16da6b42e5b39bf300c3a7cde62abb3fa3a11 can't be foun
➤ YN0061: @walletconnect/qrcode-modal@npm:1.5.2 is deprecated: All published versioned below 1.6.0 are deprecated. Please upgrade to the latest version
➤ YN0061: @walletconnect/client@npm:1.5.2 is deprecated: All published versioned below 1.6.0 are deprecated. Please upgrade to the latest version
➤ YN0061: buffer@npm:4.9.1 is deprecated: This version of 'buffer' is out-of-date. You must update to v4.9.2 or newer
➤ YN0061: querystring@npm:0.2.0 is deprecated: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
➤ YN0061: ganache-core@npm:2.13.2 is deprecated: ganache-core is now ganache; visit https://trfl.io/g7 for details
➤ YN0061: eth-sig-util@npm:3.0.0 is deprecated: Deprecated in favor of '@metamask/eth-sig-util'
➤ YN0061: querystring@npm:0.2.1 is deprecated: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
➤ YN0061: source-map-url@npm:0.4.1 is deprecated: See https://github.com/lydell/source-map-url#deprecated
➤ YN0061: source-map-resolve@npm:0.6.0 is deprecated: See https://github.com/lydell/source-map-resolve#deprecated
➤ YN0061: har-validator@npm:5.1.5 is deprecated: this library is no longer supported
➤ YN0061: request-promise-native@npm:1.0.9 is deprecated: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
➤ YN0061: safe-event-emitter@npm:1.0.1 is deprecated: Renamed to @metamask/safe-event-emitter
➤ YN0032: nan@npm:2.14.2: Implicit dependencies on node-gyp are discouraged
➤ YN0032: keccak@npm:1.4.0: Implicit dependencies on node-gyp are discouraged
➤ YN0032: keccak@npm:2.1.0: Implicit dependencies on node-gyp are discouraged
➤ YN0032: keccak@npm:3.0.1: Implicit dependencies on node-gyp are discouraged
➤ YN0032: nan@npm:2.15.0: Implicit dependencies on node-gyp are discouraged
➤ YN0032: secp256k1@npm:3.8.0: Implicit dependencies on node-gyp are discouraged
➤ YN0032: secp256k1@npm:4.0.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: axios@npm:0.18.1 is deprecated: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410
➤ YN0061: source-map-resolve@npm:0.5.3 is deprecated: See https://github.com/lydell/source-map-resolve#deprecated
➤ YN0061: ethereumjs-block@npm:1.7.1 is deprecated: New package name format for new versions: @ethereumjs/block. Please update.
➤ YN0061: ethereumjs-block@npm:2.2.2 is deprecated: New package name format for new versions: @ethereumjs/block. Please update.
➤ YN0061: ethereumjs-vm@npm:2.6.0 is deprecated: New package name format for new versions: @ethereumjs/vm. Please update.
➤ YN0061: ethereumjs-vm@npm:4.2.0 is deprecated: New package name format for new versions: @ethereumjs/vm. Please update.
➤ YN0061: ethereumjs-tx@npm:1.3.7 is deprecated: New package name format for new versions: @ethereumjs/tx. Please update.
➤ YN0061: ethereumjs-tx@npm:2.1.2 is deprecated: New package name format for new versions: @ethereumjs/tx. Please update.
➤ YN0061: object-keys@npm:0.4.0 is deprecated:
➤ YN0061: cids@npm:0.7.5 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: eth-json-rpc-errors@npm:1.1.1 is deprecated: Package renamed: https://www.npmjs.com/package/eth-rpc-errors
➤ YN0061: eth-json-rpc-errors@npm:2.0.2 is deprecated: Package renamed: https://www.npmjs.com/package/eth-rpc-errors
➤ YN0061: multibase@npm:0.6.1 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: multicodec@npm:0.5.7 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: multicodec@npm:1.0.4 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: multibase@npm:0.7.0 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: promisify-es6@npm:1.0.3 is deprecated: Unmaintained
➤ YN0061: ipld-dag-pb@npm:0.17.4 is deprecated: This module has been superseded by @ipld/dag-pb and multiformats
➤ YN0061: peer-info@npm:0.15.1 is deprecated: No longer supported for js-libp2p0.28.0 or later
➤ YN0061: ipld-raw@npm:4.0.1 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: cids@npm:0.8.3 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: multibase@npm:1.0.1 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: @walletconnect/core@npm:1.5.2 is deprecated: All published versioned below 1.6.0 are deprecated. Please upgrade to the latest version
➤ YN0061: ipld-dag-cbor@npm:0.15.3 is deprecated: This module has been superseded by @ipld/dag-cbor and multiformats
➤ YN0061: ethereumjs-common@npm:1.5.0 is deprecated: New package name format for new versions: @ethereumjs/common. Please update.
➤ YN0061: ethereumjs-common@npm:1.5.2 is deprecated: New package name format for new versions: @ethereumjs/common. Please update.
➤ YN0061: ethereumjs-account@npm:3.0.0 is deprecated: Please use Util.Account class found on package ethereumjs-util@^7.0.6 https://github.com/ethereumjs/ethereumjs-util/releases/tag/v7.0.6
➤ YN0061: @walletconnect/mobile-registry@npm:1.4.0 is deprecated: Deprecated in favor of dynamic registry available from: https://github.com/walletconnect/walletconnect-registry
➤ YN0061: @ensdomains/resolver@npm:0.2.4 is deprecated: Please use @ensdomains/ens-contracts
➤ YN0061: @ensdomains/ens@npm:0.4.5 is deprecated: Please use @ensdomains/ens-contracts
➤ YN0061: svgo@npm:1.3.2 is deprecated: This SVGO version is no longer supported. Upgrade to v2.x.x.
➤ YN0032: evp_bytestokey@npm:1.0.3: Implicit dependencies on node-gyp are discouraged
➤ YN0061: urix@npm:0.1.0 is deprecated: Please see https://github.com/lydell/urix#deprecated
➤ YN0061: resolve-url@npm:0.2.1 is deprecated: https://github.com/lydell/resolve-url#deprecated
➤ YN0032: node-addon-api@npm:2.0.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: @hapi/joi@npm:15.1.1 is deprecated: Switch to 'npm install joi'
➤ YN0061: rollup-plugin-babel@npm:4.4.0 is deprecated: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-babel.
➤ YN0061: ethereumjs-blockchain@npm:4.0.4 is deprecated: New package name format for new versions: @ethereumjs/blockchain. Please update.
➤ YN0061: sane@npm:4.1.0 is deprecated: some dependency vulnerabilities fixed, support for node < 10 dropped, and newer ECMAScript syntax/features added
➤ YN0061: protons@npm:1.2.1 is deprecated: This module is no longer maintained
➤ YN0032: bufferutil@npm:4.0.3: Implicit dependencies on node-gyp are discouraged
➤ YN0061: multihashing-async@npm:0.7.0 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: multihashing-async@npm:0.8.2 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: multihashing-async@npm:0.5.2 is deprecated: This module has been superseded by the multiformats module
➤ YN0061: testrpc@npm:0.0.1 is deprecated: testrpc has been renamed to ganache-cli, please use this package from now on.
➤ YN0032: utf-8-validate@npm:5.0.5: Implicit dependencies on node-gyp are discouraged
➤ YN0061: ethashjs@npm:0.0.8 is deprecated: New package name format for new versions: @ethereumjs/ethash. Please update.
➤ YN0061: @hapi/hoek@npm:8.5.1 is deprecated: This version has been deprecated and is no longer supported or maintained
➤ YN0061: @hapi/address@npm:2.1.4 is deprecated: Moved to 'npm install @sideway/address'
➤ YN0061: @hapi/bourne@npm:1.3.2 is deprecated: This version has been deprecated and is no longer supported or maintained
➤ YN0061: @hapi/topo@npm:3.1.6 is deprecated: This version has been deprecated and is no longer supported or maintained
➤ YN0061: flatten@npm:1.0.3 is deprecated: flatten is deprecated in favor of utility frameworks such as lodash.
➤ YN0061: core-js@npm:2.6.12 is deprecated: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
➤ YN0061: libp2p-crypto-secp256k1@npm:0.3.1 is deprecated: Included in libp2p-crypto, use it instead. https://github.com/libp2p/js-libp2p-crypto
➤ YN0061: multihashing-async@npm:0.6.0 is deprecated: This module has been superseded by the multiformats module
➤ YN0032: ursa-optional@npm:0.10.2: Implicit dependencies on node-gyp are discouraged
➤ YN0061: mkdirp-promise@npm:5.0.1 is deprecated: This package is broken and no longer maintained. 'mkdirp' itself supports promises now, please switch to that.

**Comment by metaverde** [comment]
MetaRaj, I'm on Ubuntu, and that yarn version, 3.1.1, doesn't work on my machine either.
I've had to purge yarn then npm install it to get version 1.22.17, which works beautifully.
Idk the ins, outs, and whys.

I hope this helps.

**Comment by MetaRaj** [comment]
Hi Metaverde, thank you very much for taking your time in helping me with this. Yes I managed to solve it by :

-  yarn set version classic  which gave me v1.23.0;  the only way to purge that I could think off 
- Then had many " warning "workspace-aggregator-169d132c-bb2f-432b-b555-2dd7ed147e1c " which i then used " yarn config set workspaces-experimental true " to solve
- Had hardhat run error. Found out from the Issues section here that support is only for up to Hardhat v16. So had to uninstall and reinstall.
So far all ok and looking forward to continuing with the tutorial here.
I hope my detailed explanation above could help a newbee like me save time in the scaffold-eth installation.

Deeply  grateful to to Metaverde.
